ghcup-0.0.7.yaml

```diff
@@ -3542,6 +3542,39 @@ ghcupDownloads:
             unknown_versioning:
               dlUri: https://downloads.haskell.org/~ghcup/unofficial-bindists/cabal/3.8.1.0/armv7-linux-cabal-3.8.1.0.tar.xz
               dlHash: 836d89aa1c98a3a848b8b691f9b99123f260dcd4cc1163cb77435a31559475fe
+    3.10.1.0:
+      viTags:
+        - Latest
+      viChangeLog: https://github.com/haskell/cabal/blob/master/release-notes/cabal-install-3.10.1.0.md
+      viArch:
+        A_64:
+          Linux_UnknownLinux:
+            unknown_versioning: &cabal-31010-64
+              dlUri: https://downloads.haskell.org/~cabal/cabal-install-3.10.1.0/cabal-install-3.10.1.0-x86_64-linux-alpine.tar.xz
+              dlHash: 187fa3d6e69bd8647aea121e469a1c8e4688a704ab88df54e9547c571e1cdd99
+          Linux_Alpine:
+            unknown_versioning: *cabal-31010-64
+          Darwin:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~cabal/cabal-install-3.10.1.0/cabal-install-3.10.1.0-x86_64-darwin.tar.xz
+              dlHash: 893a316bd634cbcd08861306efdee86f66ec634f9562a8c59dc616f7e2e14ffa
+          Windows:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~cabal/cabal-install-3.10.1.0/cabal-install-3.10.1.0-x86_64-windows.zip
+              dlSubdir:
+              dlHash: 31ca1cd173d4da675bc9790746d6b492cbe204e0332b282141d7ecc8ae43997b
+        A_32:
+          Linux_UnknownLinux:
+            unknown_versioning: &cabal-31010-32
+              dlUri: https://downloads.haskell.org/~ghcup/unofficial-bindists/cabal/3.10.1.0/cabal-install-3.10.1.0-i386-linux-deb9.tar.xz 
```

Comment:
This is wrong. It seems you just copy pasted the previous release?

Reply:
```suggestion
              dlUri: https://downloads.haskell.org/~ghcup/unofficial-bindists/cabal/3.10.1.0/cabal-install-3.10.1.0-i386-linux-deb9.tar.xz 
```

Reply:
Oh, yes, apologies. I probably only updated the name of the file and missed the path.